transcom/mymove
# B 21997 UI order

## [B-21977](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21997)

## Summary
This PR alters what roles are eligible for supervisor privileges. It is downstream of [B-21553](https://github.com/transcom/mymove/pull/14358)

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Access the admin application
2. Navigate to the office users tab
3. Create a new office user
4. Ensure that supervisor privileges are correct on this form
5. Complete creation of the office user
6. Edit the office user you created
7. Ensure that supervisor privileges are correct on this form
8. Save your changes
9. Optionally log into your created office user to ensure their supervisor permissions are working correctly.

### Frontend
